src/api/Jalasoft.TeamUp.Projects.API/Startup.cs

```diff
@@ -21,8 +21,9 @@ public override void Configure(IFunctionsHostBuilder builder)
             builder.Services.AddScoped<IProjectsService, ProjectsService>();
             builder.Services.AddScoped<IInvitationsService, InvitationsService>();
 #if DEBUG
-            builder.Services.AddScoped<IProjectsRepository, ProjectsInMemoryRepository>();
-            builder.Services.AddScoped<IInvitationsRepository, InvitationsInMemoryRepository>();
+            builder.Services.AddScoped<IProjectsRepository, ProjectsMongoDbRepository>();
+            builder.Services.AddScoped<IInvitationsRepository, InvitationsMongoDbRepository>();
+
```

Comment:
please double-check if is needed to make changes to this file

Reply:
Done